### DISCUSSION TITLE
How do I make my current org a nonprofit

### DISCUSSION THREAD

**Discussion by divergentadventureclub** [community/community#146683] [New to GitHub]
I have an organization and the actual underlying is a 501c3 nonprofit but when I got to nonprofit.github.com the dropdown to "validate my nonprofit" doesn't have my org in it to sign up.  How can I correct this? sign up.  How can I correct this?

**Comment by pooulad** [comment]
If you're trying to validate your nonprofit organization for GitHub's nonprofit discount program but can't find your organization listed in the dropdown on [nonprofit.github.com](https://nonprofit.github.com), here are the steps you can take:

* * *

### 1\. **Confirm Your Organization's Eligibility**

Make sure your organization qualifies for the GitHub Nonprofit Program. Generally, this requires the organization to be a verified 501(c)(3) nonprofit. Check that your nonprofit status is up-to-date with the IRS and properly recognized as a 501(c)(3).

* * *

### 2\. **Ensure Correct Organization Details**

Double-check that the name of your organization on GitHub matches exactly with the name of your nonprofit. GitHub’s validation system might rely on precise matches, and small differences in names could cause your organization to not appear in the dropdown.

* * *

### 3\. **Submit the Nonprofit Validation Form**

If your organization is not listed in the dropdown, you can manually submit a validation request to GitHub by following these steps:

-   Go to [GitHub’s Nonprofit Program Page](https://nonprofit.github.com).
-   Scroll to the bottom and look for an option to "Request validation" or "Can't find your nonprofit?" (if the dropdown is empty).
-   Follow the instructions to submit your nonprofit information for manual validation.
-   You may need to provide proof of your 501(c)(3) status, such as an IRS determination letter.
* * *

### 4\. **Contact GitHub Support**

If you still can’t get your organization validated or listed in the dropdown:

-   Go to [GitHub Support](https://support.github.com/contact) and submit a support request.
-   In your request, mention your organization's full name, nonprofit status, and a description of the issue you're facing with the dropdown.
-   Include any relevant documents or links proving your nonprofit status.
* * *

### 5\. **Check GitHub’s Terms for Nonprofits**

Review GitHub’s official [Nonprofit Program Terms](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/github-s-nonprofit-program) to ensure your organization meets all requirements.

* * *

**Comment by Rick-Reef** [comment]
I had the same issue. My Non-Profit organisation showed up in the drop down after I had authorised "Nonprofits on GitHub" access to my account. I had received an email with a long link that didn't look like anything and thus missed. However, you should be able to authorise access from your Personal Account Settings under Integrations > Applications > Authorized OAuth Apps.
Once this was done, my Organisation showed up in the drop down - https://nonprofits.github.com/ - and I was able to progress my application.
If you find out how to take advantage of the NonProfit offers, please let me know (as this is now where I am stuck).

**Reply by divergentadventureclub** [reply]
Thanks I deleted the oauth and accepted it again but nothing happened.  Maybe there's an email component to it.  I'll give it another shot.

**Reply by divergentadventureclub** [reply]
This time it worked .  There was a button to grant access this time that I didn't see last two times so I think that was the key.    

<img width="479" alt="image" src="https://github.com/user-attachments/assets/8270cb33-055e-4d1c-b100-5badf320f58a" />

I'm pending approval so once that's in I'm going to set up the team and I'll let you know if it works.

**Reply by divergentadventureclub** [reply]
I got through the dropdown and then did the approval steps including submitting my 501C3 letter.  Once I did that I got an email that said approved.  So I was able to select "teams" and then I got an email that said it's a manual process and I'd get an email when it was done.
